src/app/clusters/temperature-control-server/temperature-control-server.cpp

```diff
@@ -131,17 +130,15 @@ bool emberAfTemperatureControlClusterSetTemperatureCallback(app::CommandHandler
         {
             int16_t minTemperature = 0;
             int16_t maxTemperature = 0;
-            emberAfStatus          = MinTemperature::Get(endpoint, &minTemperature);
-            if (emberAfStatus != EMBER_ZCL_STATUS_SUCCESS)
+            status                 = MinTemperature::Get(endpoint, &minTemperature);
+            if (status != Status::Success)
```

Comment:
This can be simplified with a SuccessOrExit. This applies multiple times in the file.

Reply:
Nope